stable/spinnaker/requirements.yaml

```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: redis
-  version: 3.8.0
+  version: 10.5.3
   repository: https://kubernetes-charts.storage.googleapis.com/
   condition: redis.enabled
 - name: minio
-  version: 1.6.3
+  version: 5.0.9
```

Comment:
Since this is a massive chart change, did you address the possibility of any core changes in the mini chart? There might have been some variables moved round.

Reply:
Your right I had Minio disabled and it does not deploy (I had it turned off on mine). I committed a new defaults that worked in a test environment.

Reply:
Wanted to add to this that this is the reason I bumped the minor version number of the chart instead of the patch version, this may require a major version bump depending on what your thoughts are. People will have to update their Minio settings and possibly their Redis settings.